sunpy/util/great_arc.py

```diff
@@ -0,0 +1,121 @@
+#
+# Calculates the co-ordinates along great arcs between two specified points
+# which are assumed to be on disk.
+#
+from __future__ import absolute_import, division, print_function
+import numpy as np
+import sunpy.coordinates
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+
+
+def great_arc(a, b, center=SkyCoord(0*u.km, 0*u.km, 0*u.km, frame='heliocentric'), num=100):
```

Comment:
What does a great circle mean if it's not through the centre of the sun? You could have a sphere centred on an AR, say, and produce great circles, but then you would have another set of coordinates...